README.md

```diff
@@ -19,7 +19,16 @@
 
 ## Purpose
 
-The goal of this project is to make the easiest, fastest, and most painless way of setting up a self-hosted Git service. Using Go, this can be done with an independent binary distribution across **all platforms** which Go supports, including Linux, macOS, and Windows on x86, amd64, ARM and PowerPC architectures. Want to try it before doing anything else? Do it [with the online demo](https://try.gitea.io/)! This project has been [forked](https://blog.gitea.io/2016/12/welcome-to-gitea/) from [Gogs](https://gogs.io).
+The goal of this project is to make the easiest, fastest, and most
```

Comment:
I think we have discussed this many times. I don't think this is a good idea.

Reply:
@lunny Any reason why we should _not_ do it? From my point of view we allow people to review using `git diff` and in GitHub nothing would change.